python/lsst/gdb/afw/printers.py

```diff
@@ -1,5 +1,10 @@
 from __future__ import print_function
 from __future__ import absolute_import
+from __future__ import division
+from builtins import str
+from builtins import range
+from builtins import object
+from past.builtins import long
```

Comment:
I wanted to retain the distinction on the python2 version as I had assumed it was relatively important. That's how I did previous python3 migrations (e.g. in lsst@daf_base#8).